person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, name, age, parents_permission = 'Unknown')
+    @id = id,
+          @name = name,
+          @age = age,
+          @parents_permission = parents_permission
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+  end
+
+  def can_use_services?
+    return true if @age || @parents_permission == true
+  end
```

Comment:
- [ ] As mentioned by the previous reviewer you should just call the method `of_age` like `of_age?() || @parent_permission == true` 🙃 
- [ ] Also remember that only the `of_age` method is private, but for you here the two methods are private. To fix that you can move your method `of_age` after `can_use_services?` or before the method `can_use_services?` you can add public, to put it public 🙃 🙃